client/signup/steps/user/index.jsx

```diff
@@ -495,7 +453,7 @@ export class UserStep extends Component {
 					submitButtonText={ this.submitButtonText() }
 					suggestedUsername={ this.props.suggestedUsername }
 					handleSocialResponse={ this.handleSocialResponse }
-					isPasswordlessExperiment={ this.isPasswordlessExperiment() }
+					isPasswordless={ isMobile() }
 					experimentName={ this.state.experiment }
```

Comment:
Note that we're still using the `experiment` local state, here. Should we ditch the `experimentName` prop?

Reply:
Good catch @tyxla! Will create a PR to clean this up.

Reply:
Thanks, @enejb!